Controllers/CampaignsController.cs

```diff
@@ -92,13 +92,12 @@ public async Task<IActionResult> AddLootByTauheadURL(int campaignId, string url)
             if (model == null) return Conflict();
             return PartialView("_LootCardPartial", model);
         }
-
-        public async Task<IActionResult> Loot(int[] display = null)
+// Added tierSelect and typeSelect

```

Comment:
This message should be in commit message only, IMO - not in the code.

Reply:
I don't really know how to add commit comments for particular lines.

Reply:
Well, there is no way to add a commit message to a particular line - but as long as you just mention what you've done in the commit message, it should be enough.